gapic-common/lib/gapic/grpc/service_stub.rb

```diff
@@ -15,18 +15,24 @@
 require "grpc"
 require "googleauth"
 require "gapic/grpc/service_stub/rpc_call"
+require "gapic/grpc/service_stub/channel"
+require "gapic/grpc/service_stub/channel_pool"
+
 
 module Gapic
   ##
   # Gapic gRPC Stub
   #
-  # This class wraps the actual gRPC Stub object and it's RPC methods.
+  # This class wraps the actual gRPC Stub and ChannelPool objects.
   #
   # @!attribute [r] grpc_stub
   #   @return [Object] The instance of the gRPC stub class (`grpc_stub_class`) constructor argument.
+  # @!attribute [r] channel_pool
+  #   @return [Object] The instance of the ChannelPool class.
```

Comment:
Can the type here be documented as `ChannelPool`?

Reply:
Updated.